File: service/src/main/java/no/nav/melosys/service/dokument/brev/BrevDataAvslagArbeidsgiver.java
package no.nav.melosys.service.dokument.brev;

import java.util.Set;

import no.nav.melosys.domain.Lovvalgsperiode;
import no.nav.melosys.domain.VilkaarBegrunnelse;
import no.nav.melosys.domain.avklartefakta.AvklartVirksomhet;
import no.nav.melosys.domain.person.Persondata;


public class BrevDataAvslagArbeidsgiver extends BrevData {
    public BrevDataAvslagArbeidsgiver(String saksbehandler) {
        this.saksbehandler = saksbehandler;
    }

    public Persondata person;
    public AvklartVirksomhet hovedvirksomhet;
    public Lovvalgsperiode lovvalgsperiode;
    public String arbeidsland;

    public Set<VilkaarBegrunnelse> vilkårbegrunnelser121;
    public Set<VilkaarBegrunnelse> vilkårbegrunnelser121VesentligVirksomhet;
}

File: service/src/main/kotlin/no/nav/melosys/service/dokument/brev/BrevDataAvslagArbeidsgiver.kt
package no.nav.melosys.service.dokument.brev

import no.nav.melosys.domain.Lovvalgsperiode
import no.nav.melosys.domain.VilkaarBegrunnelse
import no.nav.melosys.domain.avklartefakta.AvklartVirksomhet
import no.nav.melosys.domain.person.Persondata


class BrevDataAvslagArbeidsgiver(
    saksbehandler: String
) : BrevData(saksbehandler = saksbehandler) {
    var person: Persondata? = null
    var hovedvirksomhet: AvklartVirksomhet? = null
    var lovvalgsperiode: Lovvalgsperiode? = null
    var arbeidsland: String? = null

    var vilkårbegrunnelser121: Set<VilkaarBegrunnelse> = emptySet()
    var vilkårbegrunnelser121VesentligVirksomhet: Set<VilkaarBegrunnelse> = emptySet()
}
